src/sonymn_int.cpp

```diff
@@ -905,7 +904,7 @@ namespace Exiv2 {
             // Ranges of models that do not support this tag
             std::string model = pos->toString();
             for (auto& m : { "DSC-", "Stellar" }) {
-                if (Util::startsWith(model, m)) {
```

Comment:
I've lost track of whether we're on C++20 yet. If we are then we could use [`std::string::starts_with`](https://en.cppreference.com/w/cpp/string/basic_string/starts_with).